### PR TITLE
Added NativeYuvContext.

### PR DESCRIPTION
**Pull request by emilwestergren**
@davidpiuva peer review

### CODE REVIEW

**Comment by davidpiuva** [comment]
I will have a look.

**Comment by davidpiuva** [comment]
A bit cryptic but looks okay.

File: source/draw/gpu/opengl/NativeYuvContext.ooc [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+		super(other)
+		this _yuvShader = OpenGLMapTransform new(slurp("shaders/yuv.frag"), this)
+	}
+	free: override func {
+		this _backend makeCurrent()
+		(this _eglBin, this _yuvShader) free()
+		super()
+	}
+	createImage: override func (rasterImage: RasterImage) -> GpuImage {
+		match(rasterImage) {
```
Comment (marcusnaslund) [inline source/draw/gpu/opengl/NativeYuvContext.ooc]: Space after `match`

**Comment by marcusnaslund** [comment]
Also, please rebase

**Comment by emilwestergren** [comment]
I am pleased to inform you that this pull request has been updated.

**Comment by marcusnaslund** [comment]
And it is my esteemed pleasure to announce the merging of this pull request, may it bring you hope and joy for the future.

**Pull request merged**